swagger.yaml

```diff
@@ -0,0 +1,159 @@
+swagger: "2.0"
+info:
+  description: ""
+  version: "1.0.0"
+  title: ""
+  termsOfService: ""
+  contact:
+    email: "apiteam@swagger.io"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "maltd-api.gov.bc.ca"
+basePath: "/v1"
+tags:
+- name: "project"
+  description: "Access to projects"
+- name: "user"
+  description: "Operations about user"
+  externalDocs:
+    description: "Find out more about our project"
+    url: "https://github.com/bcgov/maltd"
+schemes:
+- "https"
+paths:
+  /users:
```

Comment:
Do we need this endpoint? I thought it was 1-1 exact match?

Reply:
ie// is there a case where we need to list ALL users?

Reply:
We would not list all users. This endpoint I had added for the search, 

```
/users?q=akroon3r
```

which you could get the details about a user with

```
/users/akroon3r
```

Reply:
thanks for clarifying :)

Reply:
We could combine the user get and search.  There are pros and cons.  It depends on if the user get also returns all the projects they are assigned to. In this case could be slower.  Having a separate search could allow, in the future, a automatic background load of the the project details. The look up of the user should be quick.